docs/index.md

```diff
@@ -23,3 +23,15 @@ OSV-Scanner provides an officially supported frontend to the [OSV database](http
 The above all results in fewer, more actionable vulnerability notifications, which reduces the time needed to resolve them. Check out our [announcement blog post] for more details!
 
 [announcement blog post]: https://security.googleblog.com/2022/12/announcing-osv-scanner-vulnerability.html
+
+## Supported Languages
+
+- C/C++
```

Comment:
Thinking more, I think this list needs to be nuanced and point to supported dependency formats instead. Maybe we can move the list in https://google.github.io/osv-scanner/usage/#specify-lockfiles to a table, and add C/C++ as a special case with (vendored dir + submodule) as the "format"?

Reply:
@oliverchang I see your point. But I also think that there is some value in including a list like this on the homepage so people can quickly see if their project is likely to be compatible. 

I am going to add a line here about the formats that links to the supported lockfiles section. Let me know what you think of the changes.

Reply:
I agree with you that we should have this overview in the index. 

However I think having a table in the overview wouldn't be bad, e.g. 3 columns, language, ecosystem/s, lockfile/s 

Or even just 2 columns, language and lock files.